package.json

```diff
@@ -17,39 +17,40 @@
   ],
   "main": "node/index.js",
   "module": "node/module.js",
-  "typings": "render.d.ts",
+  "typings": "src/node.d.ts",
```

Comment:
Why `"typings": "src/node.d.ts"`? Why not `"typings": "src/index.d.ts"`

Reply:
Well, ignore it. `src/node.d.ts` makes more sense.

Reply:
- `index.{js,d.ts}` - common code for node and browser which exports internal `renderer` and `renderToString`
- `browser.{js,d.ts}` - client-side code which re-exports `renderToString` from index and exports `withRender` with `toString` only
- `node.{js,d.ts}` - server-side code which re-exports `renderToString` from index plus `renderToStream` and also exports `withRender` with `toString` and `toStream`

Reply:
In short, `App` is the type for one `app()` function that it will need  `withRender()` as param.

Reply:
Sounds reasonable, will you create a PR to Hyperapp repo or should I?

Reply:
Sure, I will send a PR explaining about that.

Reply:
> index.{js,d.ts} - common code for node and browser which exports internal renderer and renderToString
> browser.{js,d.ts} - client-side code which re-exports renderToString from index and exports withRender with toString only
> node.{js,d.ts} - server-side code which re-exports renderToString from index plus renderToStream and also exports withRender with toString and toStream

Great!

Reply:
Only a detail about importing.
Here some examples:

```ts
// 1. Testing:
import { renderToStream, withRender, Render, renderToString } from "../../src/node"
// 2. Actually: if I want to import node version
import { renderToStream, withRender, Render, renderToString } from "@hyperapp/render/src/node"
// 3. Purpose: move typings to root directory
import { renderToStream, withRender, Render, renderToString } from "@hyperapp/render/node"
```

Why do you think about the third option?

Reply:
Both 2 and 3 options are valid after this PR.

Reply:
fine!

Reply:
The PR was sent..